src/stream.c

```diff
@@ -1425,13 +1425,10 @@ uvc_error_t uvc_stream_get_frame(uvc_stream_handle_t *strmh,
       int err = pthread_cond_timedwait(&strmh->cb_cond, &strmh->cb_mutex, &ts);
 
       //TODO: How should we handle EINVAL?
-      switch(err){
-      case EINVAL:
-          *frame = NULL;
-          return UVC_ERROR_OTHER;
-      case ETIMEDOUT:
-          *frame = NULL;
-          return UVC_ERROR_TIMEOUT;
+      if(err){
```

Comment:
Please add spaces before/after () like the other `if`s in this file. Otherwise looks OK.

Reply:
done